management/server/types/network.go

```diff
@@ -33,6 +34,52 @@ type NetworkMap struct {
 	OfflinePeers        []*nbpeer.Peer
 	FirewallRules       []*FirewallRule
 	RoutesFirewallRules []*RouteFirewallRule
+	ForwardingRules     []*ForwardingRule
+}
+
+func (nm *NetworkMap) Merge(other *NetworkMap) {
```

Comment:
we might endup with duplicates on some key fields. We should check for unique on:
Peers
Routes
OfflinePeers